qa/tasks/cephfs/test_volume_client.py

```diff
@@ -98,7 +101,7 @@ def _configure_guest_auth(self, volumeclient_mount, guest_mount,
             vp = VolumePath("{group_id}", "{volume_id}")
             auth_result = vc.authorize(vp, "{guest_entity}", readonly={readonly},
                                        tenant_id="{tenant_id}")
-            print auth_result['auth_key']
+            print(auth_result['auth_key'])
```

Comment:
you need to ```from __future__ import print_function```

Reply:
@tchaikov I am executing these scripts/payloads using python3. So, that import is not required, right?

Reply:
are we dropping the py2 support? if yes, are you pure-py3 ready?

Reply:
Yes we need:

> you need to from __future__ import print_function

because the code must still work in py2.

Reply:
Yeah, I was working with intention to port which was wrong. I need to make py3 compatible, not port it make incompatible with py2.

Reply:
Done.

Reply:
@rishabh-d-dave you need to `from __future__ import print_function` in this source file.

Reply:
I don't get it; why? I've added it for the embedded python script [1] and there are no print statements otherwise.

[1] https://github.com/rishabh-d-dave/ceph/blob/volclient-py2-to-py3/qa/tasks/cephfs/test_volume_client.py#L27